io.github.suchnsuch.Tangent.metainfo.xml

```diff
@@ -0,0 +1,96 @@
+    <name>Taylor Hadden</name>
+    <url>https://everything-abridged.com/me/</url>
+  </developer>
+  <content_rating type="oars-1.1" />
+  <summary>A clean and powerful open source notes app</summary>
+
+  <categories>
+    <category>Office</category>
+  </categories>
+
+  <description>
+    <p>
+      Tangent is designed to let you write the way you think. Your thoughts are connected; Tangent lets you link to existing notes and make new ones with ease.
+      Let your mind wander along the path of interest. Don't suppress thoughts, embrace them. Key features include:
+    </p>
+    <ul>
+      <li>[[Wiki-link]] linking to other files in your workspace by name.</li>
+      <li>Horizontally-scrolling sliding panels for quickly moving back and forth along a trail of thoughts.</li>
+      <li>Note creation automation with support for creating notes within folders defined by wildcards.</li>
+      <li>A Map View of the links between notes created and followed during your current session.</li>
+      <li>Full-workspace note search powered by a custom query language.</li>
+    </ul>
+    <p>
+      Tangent reads and writes plaintext notes using a loose interpretation of Markdown syntax.
+      Text is rendered in a pseudo-WYSIWYG style, with formatting characters appearing only when you need them.
+      Tangent supports bold, italics, headers, lists, links, images, todos, tags, inline code, block code with syntax highlighting,
+      inline and block KaTeX math, color highlights, strikethrough, and comments.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">io.github.suchnsuch.Tangent.desktop</launchable>
+
+  <branding>
+    <color type="primary" scheme_preference="light">#55dd84</color>
+    <color type="primary" scheme_preference="dark">#1d9547</color>
+  </branding>
+
+  <recommends>
+    <memory>200</memory>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <display_length compare="ge">625</display_length>
+  </recommends>
+
+  <requires>
+    <display_length compare="ge">450</display_length>
+  </requires>
+
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </supports>
+
+  <screenshots>
+    <screenshot type="default">
+      <caption>Note view</caption>
+      <image>https://raw.githubusercontent.com/k8ieone/flathub/new-pr/screenshots/note_view.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Folder view</caption>
+      <image>https://raw.githubusercontent.com/k8ieone/flathub/new-pr/screenshots/folder_view.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Map view</caption>
+      <image>https://raw.githubusercontent.com/k8ieone/flathub/new-pr/screenshots/map_view.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Focus view</caption>
+      <image>https://raw.githubusercontent.com/k8ieone/flathub/new-pr/screenshots/focus_mode.png</image>
+    </screenshot>
```

Comment:
screenshots should be upstream not here in the PR

Reply:
I can use the screenshots from the website instead - https://www.tangentnotes.com/Features. Would that be ok?

Reply:
sure

Reply:
Alright, I'll try getting this resolved today.